packages/admin-next/dashboard/src/hooks/table/query/use-order-table-query.tsx

```diff
@@ -0,0 +1,58 @@
+import { AdminGetOrdersParams } from "@medusajs/medusa"
+import { useQueryParams } from "../../use-query-params"
+
+type UseOrderTableQueryProps = {
+  prefix?: string
+  pageSize?: number
+}
+
+/**
+ * TODO: Enable `order` query param when staging is updated
+ */
```

Comment:
comment: Currently the Order menu does nothing for the Order tables. Will enable it once staging has been updated, as it would currently crash the page.